glassfish-tck-runner/pom.template.xml

```diff
@@ -41,11 +41,24 @@
         <htmlunit.version>2.50.0</htmlunit.version>
         <shrinkwrap.version>1.2.6</shrinkwrap.version>
         <minimum.maven.version>3.0.5</minimum.maven.version>
-        <cdi.api.version>4.0.1</cdi.api.version>
-        <cdi.tck.version>4.0.6</cdi.tck.version>
+        <cdi.api.version>4.1.0</cdi.api.version>
+        <cdi.tck.version>4.1.0</cdi.tck.version>
         <httpcomponents.version>4.5.5</httpcomponents.version>
-        <arquillian.version>1.7.0.Alpha10</arquillian.version>
+        <arquillian.version>1.8.0.Final</arquillian.version>
         <jakarta.platform.version>10.0.0-RC1</jakarta.platform.version>
```

Comment:
I think that probably this property can be deleted because there is no reference for any from the dependencies